mtags/src/main/scala-3/scala/meta/internal/pc/completions/NamedArgCompletions.scala

```diff
@@ -39,6 +31,24 @@ object NamedArgCompletions:
           indexedContext,
           clientSupportsSnippets,
         )
+      case (ident: Ident) :: rest =>
+        def getApplyForContextFunctionParam(path: List[Tree]): Option[Apply] =
+          path match
+            case (app: Apply) :: _ => Some(app)
+            case (_: DefDef) :: Block(List(_), _: Closure) :: rest =>
```

Comment:
Could you add in comments when does this tree show up? A code example would be best